tensorflow/java/src/test/java/org/tensorflow/TensorMatcher.java

```diff
@@ -0,0 +1,86 @@
+/* Copyright 2016 The TensorFlow Authors. All Rights Reserved.
```

Comment:
It seems to me that we get useful information without adding more test convenience code. I'd be tempted to drop this file altogether